databricks/koalas/plot.py

```diff
@@ -992,6 +992,16 @@ def line(self, x=None, y=None, **kwargs):
         See Also
         --------
         matplotlib.pyplot.plot : Plot y versus x as lines and/or markers.
+
+        Examples
+        --------
+        Basic plot.
+
+        .. plot::
+            :context: close-figs
+
+            >>> s = ks.Series([1, 3, 2])
+            >>> ax = s.plot.line()  # doctest: +SKIP
```

Comment:
Why should we skip?

Reply:
@HyukjinKwon This doctest continuously failed on my local test bed like below:


![스크린샷 2019-09-30 오후 4 40 07](https://user-images.githubusercontent.com/23020977/65858470-fb2fd800-e3a0-11e9-81b3-23cb3427cb87.png)

But actually it works well on without exception:

<img width="495" alt="스크린샷 2019-09-30 오후 4 39 37" src="https://user-images.githubusercontent.com/23020977/65858442-ebb08f00-e3a0-11e9-8c5b-4f87354b8b26.png">

 so i wanna discussion with others members.

could you test this on your local if when available or have some insight of above failure?

Reply:
@HyukjinKwon This doctest continuously failed on my local test bed like below:


![스크린샷 2019-09-30 오후 4 40 07](https://user-images.githubusercontent.com/23020977/65858470-fb2fd800-e3a0-11e9-81b3-23cb3427cb87.png)

But it actually works well without exception:

<img width="495" alt="스크린샷 2019-09-30 오후 4 39 37" src="https://user-images.githubusercontent.com/23020977/65858442-ebb08f00-e3a0-11e9-8c5b-4f87354b8b26.png">

 so i wanna discussion with others members.

could you test this on your local if when available or have some insight of above failure?

Reply:
I just undo this skip and push again,

Reply:
Failed again. 😢 Do you have any guesswork ??

Reply:
Well i just try this doctest again with exactly same test in `test_series_plot`. But still failed.

![스크린샷 2019-09-30 오후 5 57 49](https://user-images.githubusercontent.com/44108233/65863823-d5f49700-e3ab-11e9-9879-f51e4e638b6f.png)

above passed, but below not.

![스크린샷 2019-09-30 오후 5 58 45](https://user-images.githubusercontent.com/44108233/65863903-f4f32900-e3ab-11e9-939a-3c4e1260f2b1.png)

Reply:
what if you explicitly specify `index` in Series, like `ks.Series([1, 3, 2], index=[0,1,2])`, pretty wired it directs to `timeseries` plot instead of a normal plot.

Reply:
@charlesdong1991 Thanks for comment !! I tried that way you recommend but they still throws me exception that `UNEXPECTED EXCEPTION: TypeError('index type not supported')` 😢

Reply:
and what about defining `dtype` in Series? i feel it looks like a bug to me, because I am pretty sure pandas will check the `type` of Series, and if it infers from data and it's datetime-like, then will use `timeseries` plot, otherwise, use normal plot.

Reply:
Testing only `KoalasSeriesPlotMethods.line` worked.

`./dev/pytest -k KoalasSeriesPlotMethods.line --doctest-modules databricks --verbose`

Reply:
@charlesdong1991 oh i think maybe there are some bugs in series type check either (but why this bug only reproduced in doctest?!).

i just ran test below, and it passed. 

```python
            >>> s = ks.Series(range(5),
            ...     index=pd.date_range('2018-01-01', periods=5, freq='H'))
            >>> ax = s.plot.line()
```

(and also defining `dtype` is not work neither 😢 )

Reply:
@harupy Thanks for comment!! i just try like same with you, it really is. wow it's make me very confused. @.@

Reply:
Changed the doctest as follows and it passed.
```
.. plot::
    :context: close-figs

    >>> import matplotlib.pyplot as plt
    >>> plt.close('all')
    >>> s = ks.Series([1, 3, 2])
    >>> ax = s.plot.line()
```

Reply:
@harupy oh?? then these failures before we got maybe conflicted with other existing plots??

Reply:
emm, yes, indeed it could pass tests, but for users who read the documentation, I dont know if this will bring them some confusion, so I have mixed feeling about this trick.

Reply:
Anyway, it works! Thanks ! 😸

Reply:
`close-figs` doesn't seem to close figures.
```
995 
996         Examples
997         --------
998         Basic plot.
999 
1000         .. plot::
1001             :context: close-figs
1002 
1003             >>> import matplotlib.pyplot as plt
1004             >>> plt.get_fignums()
Expected nothing
Got:
    [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21]

/Users/harutaka/Desktop/projects/koalas/databricks/koalas/plot.py:1004: DocTestFailure
```

Reply:
@charlesdong1991 yeah i totally agree with it. I think i should be find better solution for solve this issue to keep consistency of docs.
@harupy you're right i think maybe close-figs doesn't work properly either.

Reply:
Does `close-figs` has any effect when running tests?

Reply:
@harupy Oops.. sorry i missed your reply
 Anyway, yes it seems have any effect, because they are showing same result when i remove all `close-figs` options.